core/src/test/java/bio/terra/pearl/core/service/kit/LivePepperDSMClientTest.java

```diff
@@ -130,13 +116,86 @@ public void test5xxResponseFromPepper() throws Exception {
         assertThat(pepperException.getMessage(), containsString(errorResponseBody));
     }
 
+    @Transactional
+    @Test
+    public void testSendKitRequest() throws Exception {
+        // Arrange
+        PepperKitStatus kitStatus = PepperKitStatus.builder()
+                .kitId(kitRequest.getId().toString())
+                .currentStatus("New")
+                .build();
+        mockPepperResponse(HttpStatus.OK, objectMapper.writeValueAsString(kitStatus));
+
+        // Act
+        var response = client.sendKitRequest(enrollee, kitRequest, address);
+
+        // Assert
+        assertThat(response, containsString(kitStatus.getKitId()));
+        assertThat(response, containsString("New"));
+        verifyRequestForPath("/shipKit");
```

Comment:
The addition of this line is the only change to the body of this test (renamed from `testSuccessResponseFromPepper`).